components/motor/gpio/motor_encoder_test.go

```diff
@@ -349,7 +345,7 @@ func TestMotorEncoder1(t *testing.T) {
 		wg.Wait()
 
 		test.That(t, ctx.Err(), test.ShouldNotBeNil)
-		test.That(t, motorDep.state.goalRPM, test.ShouldEqual, 0)
+		test.That(t, motorDep.state.regulated, test.ShouldBeFalse)
 	})
```

Comment:
this test hangs and times out now that new context is added. will figure this out tomorrow